maven-model-builder/src/main/java/org/apache/maven/model/interpolation/StringVisitorModelInterpolator.java

```diff
@@ -1424,7 +1420,7 @@ private void visit( Properties properties )
                     {
                         String value = (String) v;
                         String inter = interpolate( value );
-                        if ( value != inter )
+                        if ( value != inter && Objects.nonNull( inter ) )
```

Comment:
`nonNull` is used in Lambda with Method Reference and not like this.
That\s the reason why `Objects.nonNull` was introduced in Java 1.8.
Simple is better. Therefore `!= null`.